packages/react-reconciler/src/ReactCurrentFiber.js

```diff
@@ -80,9 +104,22 @@ export function getCurrentFiberStackInDev(): string {
   return '';
```

Comment:
You can rewrite this on top of `getCurrentFiberStackFramesInDev` now

Reply:
after studying this a bit, I think this suggestion would cause additional duplication because i would need to pull in the work done by `getStackByFiberInDevAndProd` and there are other modules that depend on `getStackByFiberInDevAndProd`. So instead of doing a rewrite of `getCurrentFiberStackInDev` I will do a rewrite of `getStackByFiberInDevAndProd` and just get `getCurrentFiberStackInDev` to rely on it.